src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -574,21 +575,41 @@ def handleSpecialCERNMergeSettings(self, funcName):
         Enable lazy-download for fastCloning for all CMSSW_7_5 jobs (currently off)
         Enable lazy-download for all merge jobs
         """
-        cmsswVersion = os.environ['CMSSW_VERSION']
+        cmsswVersion = os.environ.get('CMSSW_VERSION', "")
         if cmsswVersion.startswith("CMSSW_7_5") and False:
```

Comment:
Hi Dirk, I know you didn't make change on this. Could you explain me again why this condition (always False) need to be set.